src/InternPool.zig

```diff
@@ -688,6 +688,18 @@ pub const Key = union(enum) {
             return @ptrCast(&ip.extra.items[self.extra_index + flags_field_index]);
         }
 
+        /// The returned pointer expires with any addition to the `InternPool`.
+        pub fn size(self: @This(), ip: *InternPool) *u32 {
+            const size_field_index = std.meta.fieldIndex(Tag.TypeUnion, "size").?;
+            return @ptrCast(&ip.extra.items[self.extra_index + size_field_index]);
```

Comment:
This is already a `u32` pointer.
```suggestion
            return &ip.extra.items[self.extra_index + size_field_index];
```